src/execution/execute.ts

```diff
@@ -511,10 +514,15 @@ export function collectFields(
         ) {
           continue;
         }
+        const updatedSelectionSet = selectionSetWithFragmentArgumentsApplied(
```

Comment:
Can't be `selectionSet`, as that shadows the function's `selectionSet`, but it can be `selectionSetWithFragmentArgumentsApplied` or something similar.